tensorflow/buildspec-tf1.yml

```diff
@@ -3,7 +3,7 @@
   framework: &FRAMEWORK tensorflow
   short_version: &SHORT_VERSION 1.15
   version: &VERSION 1.15.4
-  cuda_version: &CUDA_VERSION cu110
+  cuda_version: &CUDA_VERSION cu100
```

Comment:
Will you be reverting all the changes to buildspec-tf1.yml when merging this PR?